Globalvar.f90

```diff
@@ -77,18 +77,24 @@ module globalvar
   integer,save::flag_bnd(6)
 
 
-!for mpi 
-  integer,save::flag_mpi,flag_mpi_split,neighbor(6),my_rank,total_prc 
+!for mpi
+  integer,save::flag_mpi,flag_mpi_split,neighbor(6),my_rank,total_prc
   integer,save::mpi_siz(3),mpi_pos(3),vmpi
-  character*4 cno  
+  character*4 cno
+! for parallel HDF5 writing & reading
```

Comment:
These are the variables used in multiple subroutines for HDF5 reading and writing.